src/main/java/meteordevelopment/meteorclient/systems/System.java

```diff
@@ -49,7 +49,7 @@ public void save(File folder) {
         }
     }
 
-    public void save() {
+    public synchronized void save() {
```

Comment:
I don't think synchronization is needed here because the map is now already synchronized.

Reply:
Removed